### PR TITLE
Entra Rebrand

### PR DESCRIPTION
**Pull request by DidunAyodeji**
Updating all Azure AD terms to the Microsoft Entra ID terms in all md files.

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
The updated changes to Microsoft Entra ID terminology make sense.

## Other Information
Here's a link to the Rebrand guidelines. Note that the script doesn't have all the entries in the glossary that mine has. Let me know if you'd like to see my version of the script. https://review.learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad?branch=main

### CODE REVIEW

File: README.md
Comment (rayluo) [inline README.md:1]: This file is currently shown as entire file being changed. This is probably an issue caused by [this situation](https://stackoverflow.com/questions/12876350/diff-returning-entire-file-for-identical-files). Perhaps you can adjust your line endings in your IDE and commit again?

**Review by medhir — APPROVED** [review]
Seems like the issue @rayluo brought up is still there where the entire README is changed, also merge conflicts to be cleaned up. Otherwise looks good to me.

File: README.md
Comment (DidunAyodeji) [reply README.md:1]: Fixed this in a new PR. https://github.com/Azure-Samples/ms-identity-python-webapp/pull/149

**Comment by rayluo** [comment]
This PR was replaced by #149

**Pull request closed without merging**